### PR TITLE
Sub pages and content management

### PR DESCRIPTION
**Pull request by bradparker**
Currently on [staging](https://everydayhero.github.io/great-us-crossing/)

## Step 1: Getting access

Get a [Github Account](https://github.com/join?source=header-home), and then send me your username.

## Step 2: Editing files

Go to the [main page for this branch](https://github.com/everydayhero/great-us-crossing/tree/bp-sub-pages-and-content-management) and click through to source > content.

You should see a folder for each charity, you can edit the content of each Charity's `about.md` and `hero.json`. The `about.md` file will change the about section text and call to action, the `hero.json` file will change the hero (top of the page) section's text, logo and call to action.

To make an edit: when on a file, such as [the about section copy for MSAA](https://github.com/everydayhero/great-us-crossing/tree/master/source/content/featured-charities/msaa/about.md)

screen shot 2016-03-21 at 10 33 27 pm

Click on the pencil icon to the top right of the file preview:

screen shot 2016-03-21 at 10 33 31 pm

And edit away.

## Step 3: Saving edits

When you're happy with the changes you've made (you can keep making more after these, well get to that in a tick) scroll down to the "Commit changs" box:

screen shot 2016-03-21 at 10 36 40 pm

You may optionally write a message about the changes you've made.

**NB** just to start with, please ensure the "Create a new branch" radio is checked, as in the above example. See also that you can name your group of changes, here they're all going to be grouped under "marssie-example".

When you make a new PR like this, I'll get a notification and can check it over before deploying it.

## Step 4: Feeling comfortable with Markdown

There's lots of info over [here](https://daringfireball.net/projects/markdown/)

Markdown is great, you won't need to use a heap of what it can do in these content blocks, but in case you want `*italics*`, `**emphasis**`, or `[links](link.com)`, you can!

## Lastly: You might want to add a new charity

On the main repo page click new file and type `source/content/featured-charities/SLUG_OF_CHARITY`, replacing `SLUG_OF_CHARITY`with the new charity name you'd like to add. Please use dashes in place of spaces and all lowercase.

Copy the `about.md` and `hero.json` files from another charity folder, this might be kinda cumbersome through the github interface. Edit them as you like. You might also like to upload the logo. You should be able to drag and drop an image from your computer onto the new folder you made for the charity.

The last step is to add the charity to the `source/content/featured-charites.json` file. Copy one of the other charities in the list and edit the fields. Make sure that the link contains the name of the folder you added: `/featured-charities/SLUG_OF_CHARITY `, again replacing `SLUG_OF_CHARITY` with the 'dasherized' charity name.

Cheers and happy editing!

### CODE REVIEW

**Comment by dansowter** [comment]
Mate this is all kinds of awesome. Great work.

File: build/tasks/bundle.js [hunk 1]
```diff
@@ -17,9 +21,11 @@ const SOURCE_DIR = config.SOURCE_DIR
 const DEV_DIR = config.DEV_DIR
 const SERVER_APP_DIR = config.SERVER_APP_DIR
 
+const isDev = !process.env.NODE_ENV || (process.env.NODE_ENV === 'development')
```
Comment (alexander-bobin) [inline build/tasks/bundle.js]: Can see a `isDev` also in `source/store/actions/fetchCharity/index.js` and something similar in `build/tasks/content.js`. Maybe a helper?

Comment (bradparker) [inline build/tasks/bundle.js]: Yep. That's a good call.

File: source/layouts/Header/styles.css [hunk 2]
```diff
@@ -1,13 +1,32 @@
-.base {
-  font-size: 0.8em;
-  position: relative;
-  padding: 1em;
+.FixedHeader {
```
Comment (alexander-bobin) [inline source/layouts/Header/styles.css]: Should it be `fixedHeader`? Won't stop merge

**Pull request merged**